dubbo-common/src/main/java/org/apache/dubbo/common/utils/HttpUtils.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.dubbo.common.utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.apache.dubbo.common.utils.HttpUtils.HttpMethod.GET;
+import static org.apache.dubbo.common.utils.HttpUtils.HttpMethod.POST;
+import static org.apache.dubbo.common.utils.HttpUtils.HttpMethod.DELETE;
+import static org.apache.dubbo.common.utils.HttpUtils.HttpMethod.PUT;
+import static org.apache.dubbo.common.utils.HttpUtils.HttpMethod.HEAD;
+import static org.apache.dubbo.common.utils.HttpUtils.HttpMethod.PATCH;
+import static org.apache.dubbo.common.utils.HttpUtils.HttpMethod.OPTIONS;
+import static org.apache.dubbo.common.utils.HttpUtils.HttpMethod.TRACE;
+
+/**
+ * for http methods
+ */
+public class HttpUtils {
```

Comment:
This class is not suitable for the dubco-common module in terms of function and methods. There is no current intersection between qos and http handling except HttpMethod, and the methods in HttpUtils are not suitable for all modules.

Reply:
ok , I will change the position